paf-mvp-core-js/src/crypto/signing-definition.ts

```diff
@@ -28,16 +30,32 @@ export interface SigningDefinition<T, U = Partial<T>> {
    * @param data
    */
   getSignerDomain(data: T): string;
-
-  /**
-   * How to get input string from unsigned data
-   * @param data
-   */
-  getInputString(data: U): string;
 }
 
 export const SIGN_SEP = '\u2063';
 
+export interface SeedSignatureContainer {
+  seed: UnsignedData<Seed>;
+  idsAndPreferences: IdsAndPreferences;
+}
+
+export class SeedSignatureBuilder implements SignatureStringBuilder<SeedSignatureContainer> {
+  getInputString(data: SeedSignatureContainer): string {
+    const seed = data.seed;
+    const ids = data.idsAndPreferences.identifiers;
+    const prefs = data.idsAndPreferences.preferences;
+
+    const array = new Array<string>();
```

Comment:
[NIT] Of course this is debatable, but I think this version is more readable because much more concise:

```javascript
const array = [
  seed.source.domain,
  seed.source.timestamp.toString(),
  ...seed.transaction_ids,
  seed.publisher,
  ...ids.map((i) => i.source.signature),
  prefs.source.signature
];
```

Reply:
Done. Sorry to mention it again but having the type declaration for containers is always good in such cases. ;-)